frontend/src/app/components/about/about.component.html

```diff
@@ -134,15 +134,12 @@ <h3 i18n="about.sponsors.enterprise.withRocket">Enterprise Sponsors 🚀</h3>
         <img class="image" src="/resources/profile/bitkey.svg" />
         <span>Bitkey</span>
       </a>
-      <a href="https://bullbitcoin.com/" target="_blank" title="Bull Bitcoin">
-        <svg aria-hidden="true" class="image" viewBox="0 -5 40 40" xmlns="http://www.w3.org/2000/svg">
-          <g clip-path="url(#a)" fill-rule="evenodd" clip-rule="evenodd" fill="#e21924">
-            <path d="M21.92 14.59a1.18 1.18 0 0 0-1.18-1.18h-1.82v2.36h1.82a1.18 1.18 0 0 0 1.18-1.18ZM21 17.07h-2v2.45h2a1.23 1.23 0 1 0 0-2.45Z"/>
-            <path d="M36.43 0 35 5.59l-8 2.64-2.43-3.61-4.74 2.05-4.74-2.05-2.43 3.61-8-2.64L3.21 0 0 7.86l7.89 5.86-5.56 4 5.56 1.12 2.69-.49v3.17l3.59 4.38.68 3.19 5 2.87 5-2.87.68-3.19 3.59-4.38v-3.17l2.7.49 5.56-1.12-5.56-4 7.89-5.86zM24.69 18.45a2.5 2.5 0 0 1-2.5 2.5h-1.11v1.56h-1.26V21h-.9v1.56h-1.27V21H15.3v-1.42h.64a.9.9 0 0 0 .9-.9V14.3a.901.901 0 0 0-.9-.91h-.64V12h2.35v-1.5h1.27V12h.9v-1.5h1.26V12h.68A2.269 2.269 0 0 1 24 14.31a2.25 2.25 0 0 1-.92 1.82 2.52 2.52 0 0 1 1.58 2.32z"/>
-          </g>
-          <defs>
-            <clipPath id="a"><path fill="#fff" d="M0 0h160v32H0z"/></clipPath>
-          </defs>
+      <a href="https://bullbitcoin.com/" target="_blank" title="Bull Bitcoin" class="override-boundary">
+        <svg width="80" version="1.1" viewBox="0 50 1128 952.59" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Make the view box taller, so the `path` are not cut off

```suggestion
        <svg width="80" version="1.1" viewBox="0 0 1128 1052.59" xmlns="http://www.w3.org/2000/svg">
```